content/en/hosting/analytics/setup-docker-compose.md

```diff
@@ -19,6 +19,7 @@ This guide will walk you through setting up a deployment of CHT Sync with the CH
 - [Current version](https://docs.docker.com/engine/install/) of `docker` or current version of [Docker Desktop](https://www.docker.com/products/docker-desktop/) both of which include `docker compose`. Note that the older `docker-compose` is [no longer supported](https://www.docker.com/blog/announcing-compose-v2-general-availability/).
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 - [cht-sync](https://github.com/medic/cht-sync) GitHub repository (can be cloned via `git clone https://github.com/medic/cht-sync`).
+- [A dbt project](https://docs.getdbt.com/docs/build/projects).
```

Comment:
nitpick: IMHO it would be better to link to our docs about building the models (which in turn already link to dbt docs).

```suggestion
- [A dbt project]({{< relref "hosting/analytics/building-dbt-models" >}}).
```

Reply:
Also, it is probably worth adding a similar link over in the `setup-kubernetes.md` file.